transformers/events/vow_file/auction_attributes/transformer.go

```diff
@@ -26,28 +27,39 @@ import (
 
 type Transformer struct{}
 
-func (Transformer) ToModels(_ string, logs []core.EventLog, _ *postgres.DB) ([]event.InsertionModel, error) {
+func (Transformer) ToModels(_ string, logs []core.EventLog, db *postgres.DB) ([]event.InsertionModel, error) {
 	var models []event.InsertionModel
 	for _, log := range logs {
-		err := shared.VerifyLog(log.Log, shared.FourTopicsRequired, shared.LogDataRequired)
-		if err != nil {
-			return nil, err
+		verifyLogErr := shared.VerifyLog(log.Log, shared.FourTopicsRequired, shared.LogDataRequired)
+		if verifyLogErr != nil {
+			return nil, verifyLogErr
+		}
+
+		msgSenderAddress := common.HexToAddress(log.Log.Topics[1].Hex()).Hex()
+		msgSenderAddressId, msgSenderAddressErr := shared.GetOrCreateAddress(msgSenderAddress, db)
+		if msgSenderAddressErr != nil {
+			return nil, msgSenderAddressErr
```

Comment:
The error is supposed to be wrapped in a `ErrCouldNotCreateFK` error like so `return nil, shared.ErrCouldNotCreateFK(msgSenderErr)`

I wonder if we could make `GetOrCreateAddress` since we are doing that everywhere.